packages/custom-webpack/examples/full-cycle-app/tsconfig.app.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./out-tsc/app",
-    "types": []
+    "types": [ "node" ]
```

Comment:
I understand that this is necessary for `ts-node` to work? If yes, maybe we can put it into the hardcoded `compileOptions` when registering the `ts-node`?  
Alternatively the docs should mention that one has to add this to the config when working with `.ts` versions of `customWebpackConfig` or `indexTransform`.